enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/roles/Candidate.java

```diff
@@ -71,7 +71,7 @@
                 }
 
                 outcome.setNextRole( FOLLOWER );
-                outcome.addOutgoingMessage( new RaftMessages.Directed<>( ctx.myself(), req ) );
+                Appending.handleAppendEntriesRequest( ctx, outcome, req );
```

Comment:
Same for this one I guess, no need to have the Appending class. Just put the functionality in Follower?